pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -600,7 +600,7 @@ let
                 callPackage = newScope (scope // this.pkgs);
               };
             in
-            import ./ext.nix newSelf newSuper;
+            (import ./ext.nix newSelf newSuper) // { inherit (newSuper) callPackage; };
```

Comment:
I think I'd prefer an `inherit (super) callPackage` in `ext.nix`, because that would prevent silent overriding *if* it should be defined there as well (?)